### ISSUE TITLE
[Feature Request] This Year in Life

### ISSUE THREAD

**Issue by bungadrum** [zhrexl/ThisWeekInMyLife#1]
Is it possible to add a year option as well with a 12 month calendar layout? similar to what the older Gnome calendar had before the year option was killed off after porting to GTK 4

**Comment by zhrexl** [comment]
I think so. Can you give me an use case example?

I'm adding the option to include/remove columns. Its already possible to rename columns. With the latest gtk4 update I'll be able to change a few widgets and in the process allow for a more customizable interface.

**Comment by bragefuglseth** [comment]
This seems like something that deserves its own app. Simplicity is one of GNOME's [core design principles](https://developer.gnome.org/hig/principles.html):

> Resist the pull to try and make an app that suits all people in all situations. Focus on one situation, one type of experience.

For now, I think we should focus on developing a *great* kanban-style week planner for scheduling day-to-day activities, and not yet another calendar app that coincidentally also has a kanban board.

Just to be clear, I'm not saying that this doesn't have any use cases, but it seems a little out of scope for this specific app.

**Comment by bungadrum** [comment]
Fair enough it's just a suggestion anyway since most planners and calendars don't have a year planner option at all  some prefer to plan ahead beyond a week or month.
